clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -63,7 +63,7 @@ class AutoScalingWorker {
   private Integer healthCheckGracePeriod
   private String healthCheckType
   private String spotPrice
-  private Collection<String> suspendedProcesses
+  private Set<String> suspendedProcesses
```

Comment:
target/master and your branch both rebased against upstream I bet. I noticed that open PRs don't handle that correctly.